src/main/java/seedu/address/logic/parser/AddLessonCommandParser.java

```diff
@@ -50,9 +50,9 @@ public static Lesson parseLesson(String args, boolean nameIsOptional) throws Par
         }
     }
     private static String getUsageInfo() {
-        return "\nUsage: addLesson -name [NAME] (any number of -[subject|day|start|end|remark] [value]). "
-                + "\n For example, addLesson -name John -subject English -day 23 -start 14:30 -end 16:30"
-                + "\n If you are currently displaying lesson list, you could use 'add' inplace of 'addLesson'. "
+        return "\nUsage: addLesson -name [NAME] (any number of unique -[subject|day|start|end|remark] [value]). "
+                + "\n For example, addLesson -name John -subject Math -day 23 -start 14:30 -end 16:30"
```

Comment:
I think Math has to be Mathematics in order for the subject to be valid.

Reply:
Thanks.
Fixed in new Pr